### DISCUSSION TITLE
How to install Grammarly language server

### DISCUSSION THREAD

**Discussion by officialrajdeepsingh** [LazyVim/LazyVim#756] [Q&A]
I cannot find The right solution to install the [Grammarly language server](https://github.com/znck/grammarly) with lazyvim. I try following the solution, but it is not working.

```lua

//  plugins/grammarly.lua

require('lspconfig').grammarly.setup{
   init_options = { clientId = 'client_********HvaS', },
}

```

**Comment by dpetka2001** [comment]
To install it you can simply pass it to `nvim-lspconfig.opts.servers` or `mason.opts.ensure_installed`. Your `require` function, I think it should go to `nvim-lspconfig.opts.setup`.
